build/ci-variables.yml

```diff
@@ -0,0 +1,8 @@
+variables:
+    ResourceGroupRegion: 'southcentralus'
+    resourceGroupRoot: 'msh-fhir-ci'
+    appServicePlanResourceGroup: '$(resourceGroupRoot)-linux'
+    appServicePlanName: '$(resourceGroupRoot)-linux'
+    DeploymentEnvironmentName: '$(resourceGroupRoot)-d'
```

Comment:
What is `-d` for?

Reply:
It's short for docker. I couldn't have it in the same resource group as the current CI resources, so I needed to add something to the name.

Reply:
Let's rename it to replace the old ones